src/Common/Properties/Resources.resx

```diff
@@ -1473,8 +1473,8 @@ The file needs to be a PowerShell script (.ps1 or .psm1).</value>
 The data is anonymous and does not include commandline argument values.
 The data is collected by Microsoft.
 
-Use the Disable-AzureRmDataCollection cmdlet to turn the feature Off. The cmdlet can be found in the AzureRM.Profile module.  To disable data collection: PS &gt; Disable-AzureRmDataCollection.
-Use the Enable-AzureRmDataCollection cmdlet to turn the feature On. The cmdlet can be found in the AzureRM.Profile module.  To enable data collection: PS &gt; Enable-AzureRmDataCollection.</value>
+Use the Disable-AzureRmDataCollection cmdlet to turn the feature Off. The cmdlet can be found in the AzureRM.Accounts module.  To disable data collection: PS &gt; Disable-AzureRmDataCollection.
```

Comment:
Disable-AzDataCollection, Az.Accounts

Reply:
I *specifically stated* in the PR description that AzureRM -> Az would be done at a later date (to preserve consistency). But, I still updated this anyway to address the comment.